openff/benchmark/utils/1-validate_and_assign_ids.py

```diff
@@ -0,0 +1,54 @@
+import click
+from openforcefield.topology import Molecule
+from openforcefield.utils.toolkits import RDKitToolkitWrapper, AmberToolsToolkitWrapper, ToolkitRegistry
+
+
+@click.command()
+@click.option('-m',
+              '--molecules',
+              help='SDF file to read containing input molecules.')
+@click.option('--groupname',
+              help='Group name for assigning IDs to the molecules.')
+def validate_and_assign(molecules, groupname):
+    """
+    Load a molecule dataset from SDF, validate it for common 
+    issues, and assign it unique identifiers.
+    """
+
+    tkr = ToolkitRegistry([RDKitToolkitWrapper, AmberToolsToolkitWrapper])
+    
+    # TODO: Raise error if bad input format here?
+    
+    mols = Molecule.from_file(molecules)
```

Comment:
Support either single 3D SDF file, or directory of 3D SDFs? Latter would be more in-line with our "one molecule per file" philosophy.